src/content/energy-consumption/index.md

```diff
@@ -7,33 +7,33 @@ sidebar: true
 
 # Ethereum energy consumption {#introduction}
 
-Ethereum's current energy expenditure is too high and unsustainable. Resolving energy expenditure concerns without sacrificing security and decentralization is a significant technical challenge and has been a focus of research and development for years. Let's explore why building Ethereum has had a high environmental impact and how upcoming network upgrades will dramatically change this.
+Ethereum's current energy expenditure with [Proof of Work](/developers/docs/consensus-mechanisms/#proof-of-work) is too high and unsustainable. Resolving energy expenditure concerns without sacrificing security and decentralization is a significant technical challenge and has been a focus of research and development for years. Let's explore why building Ethereum has had a high environmental impact and how the upcoming network upgrade to [Proof of Stake](/developers/docs/consensus-mechanisms/pos/#top) will dramatically change this.
```

Comment:
Small tweak as per [the site convention](https://ethereum.org/en/contributing/style-guide/#proof-of)

```suggestion
Ethereum's current energy expenditure with [proof-of-work](/developers/docs/consensus-mechanisms/#proof-of-work) is too high and unsustainable. Resolving energy expenditure concerns without sacrificing security and decentralization is a significant technical challenge and has been a focus of research and development for years. Let's explore why building Ethereum has had a high environmental impact and how the upcoming network upgrade to [proof-of-stake](/developers/docs/consensus-mechanisms/pos) will dramatically change this.
```